### ISSUE TITLE
Reproduce gender from text classification model

### ISSUE THREAD

**Issue by glorisonne** [davidjurgens/support#1]
I'd like to reproduce the gender from text classification model described in section 4.2 of the paper to apply it to Reddit posts in my research project.
I was wondering whether there are more detailed instructions available on how to do this with this codebase in addition to the main README.

In particular I've got the following questions:
Is the training data for this model available somewhere?
Gender labels do not seem to be included in data/training-data.with-ids.tsv.
I wasn't able to locate the "example.sh script in the main directory" referenced in the Introduction section of the README.

Thank you for making this code available and I'm looking forward to your support.
